openeo/udf/udf_signatures.py

```diff
@@ -85,3 +86,16 @@ def apply_metadata(metadata: CollectionMetadata, context: dict) -> CollectionMet
 
     """
     pass
+
+
+def apply_vectorcube(geometries: any, cube: xarray.DataArray, context: dict) -> (any, xarray.DataArray):
```

Comment:
`any` is a function, I guess you meant  `typing.Any` here.

But regardless, `Any` is way to liberal, I would at least put a GeoDataFrame annotation in string format

Reply:
Added the annotation in string format.